visualization/app/codeCharta/ui/codeMap/threeViewer/threeSceneService.ts

```diff
@@ -35,6 +35,8 @@ export class ThreeSceneService implements CodeMapPreRenderServiceSubscriber, Map
 	mapGeometry: Group
 	private readonly lights: Group
 	private mapMesh: CodeMapMesh
+	/** Temporary workaround until mapMesh can be derived from store */
```

Comment:
The TODO Prefix will highlight this comment in IDEs :-)
```suggestion
	/** TODO: Fix temporary workaround as soon as mapMesh can be derived from store */
```